fre/cmor/cmor_mixer.py

```diff
@@ -56,22 +57,24 @@ def copy_nc(in_nc, out_nc):
     dsout.close()
 
 
-def get_var_filenames(indir, var_filenames = None):
+def get_var_filenames(indir, var_filenames = None, local_var = None):
     '''
-    appends files ending in .nc located within indir to list var_filenames accepts two arguments
+    appends files ending in .nc located within indir to list var_filenames accepts three arguments
         indir: string, representing a path to a directory containing files ending in .nc extension
         var_filenames: list of strings, empty or non-empty, to append discovered filenames to. the
                        object pointed to by the reference var_filenames is manipulated, and so need
                        not be returned.
+        local_var: string, optional, if not None, will be used for ruling out filename targets
```

Comment:
What is the purpose of this `local_var` setting? Is it related to the optional name renaming (e.g. t_ref to tas)?